web/src/SidebarPinButton.tsx

```diff
@@ -30,8 +30,7 @@ let UnpinnedPinIcon = styled(PinResourceFilledSvg)`
   opacity: 0;
 
   .u-showPinOnHover:hover &,
-  ${PinButton}:focus &,
-  ${PinButton}.u-persistShow & {
```

Comment:
why did the persistShow stuff go away? it seems weird that a PR about trigger mode has a bunch of changes to pinning

Reply:
Ah yes, sorry about this! I didn't do this carefully enough.

Based on designs that Surbhi and I were working on, I ended up moving the pin position to make room for the new Auto button (it won't fit in the Sidebar otherwise, since pin is also on the bottom right). So I was playing around with the classes to try to understand what `u-persistShow` does, but I didn't properly clean up after myself.

Can you explain what `u-persistShow` does, actually? I had thought it was necessary because buttons on the BuildBox at the bottom sort of gets covered up by the Popover without this class. But now I'm not so sure if that's what it does.

Reply:
i think there were a bunch of iterations where it showed up differently in different contexts. to be clear, i'm not opposed to doing a cleanup pass on the pinning stuff, i was mainly surprised to see it tacked on to an unrelated PR.

Reply:
Gotcha! I reverted the `persistShow` changes but kept the change where I moved the button.